client/src/plus/icon-card/index.scss

```diff
@@ -10,8 +11,9 @@
   border: 1px solid var(--border-secondary);
   border-radius: var(--elem-radius);
   background-color: var(--background-primary);
-  padding: 1rem;
+  padding: 1rem 1rem 1rem 0.5rem;
   transition: 0.3s ease-out; // this removes a flash from .doomed animations
+  animation: fade-in 0.2s;
```

Comment:
If we need this, we can integrate it into the current SASS based on the one in `minimalist`